readme.md

```diff
@@ -85,10 +96,15 @@ article: {
 }
 ```
 
+In the above example, the content will be:
+
+```text
+Bru is a simple markup language with json like semantics.
+It's currently used in Bruno to save details of an api request in a file.
+```
+
 ### Annotations
-Annotations are used to provide additional information about a key-value pair.
-An annotation starts with `@` and ends with a newline `\n`.
-Optionally, Comma seperated args can be passed to an annotation.
+Annotations are used to provide additional information about a key-value pair. An annotation starts with (`@`) and ends with a newline (`\n`). Arguments may be passed to an annotation in parentheses (`()`) and separated by commas. Argument values may *only* be primitive types.
```

Comment:
Making assumptions here, but there is no indication that annotation with arguments could span multiple lines. Should they be able to do so?

Reply:
IMO, Although we could potentially have the lang allow composite types as args to annotations, we should keep it simple right now and just have **only** support primitive types as args

In the future, if a strong need arises, then it should be possible to extend it like below.
It would be by allowing args to be delimited by newlines. 
```
{
  query: {
    @description('The status of the user')
    @enum(
      {
        name: 'active'
        description: 'The user is active'
      }
      {
        name: 'inactive'
        description: 'The user is inactive'
      }
    )
    status: active
  }
}
```